pkg/RepoManager.go

```diff
@@ -52,24 +52,26 @@ func (impl *HelmRepoManagerImpl) LoadIndexFile(chartRepo *sql.ChartRepo) (*repo.
 	return index, nil
 }
 
-func (impl *HelmRepoManagerImpl) ValuesJson(baseurl string, version *repo.ChartVersion) (rawValues string, readme string, err error) {
+func (impl *HelmRepoManagerImpl) ValuesJson(baseurl string, version *repo.ChartVersion) (rawValues string, readme string, schemaJson string, notes string, err error) {
+	impl.logger.Infow("snehith values json")
```

Comment:
can remove/update this log

Reply:
added logs for testing, will remove them when I raise for PR